docs/reference/operator.md

```diff
@@ -619,7 +634,9 @@ The `flatten` operator flattens each item from a source channel that is a list o
 
 As shown in the above example, deeply nested collections are also flattened.
 
-See also: [flatMap](#flatmap)
+:::{tip}
+As a best practice, it is recommended to use [`flatMap`](#flatmap) instead of `flatten`, since `flatMap` only flattens one level and has a well-defined return type.
```

Comment:
```suggestion
Use [`flatMap`](#flatmap) instead of `flatten` to apply a mapping function. `flatMap` only flattens one level and has a well-defined return type.
```
I don't think I captured this properly. But it would be good to explain why this is recommended.